block_data/heartbeat.tex

```diff
@@ -44,12 +44,12 @@
 We now impose constraints on $\relBlock$.
 \begin{enumerate}[resume]
 	\item $\relBlock _{0} = 0$
-	\item $\relBlock_{i + 1} \in \{ \relBlock_{i}, 1 + \relBlock_{i} \}$ \quad (\sanityCheck)
+	\item $\relBlock _{i + 1} \in \{ \relBlock_{i}, 1 + \relBlock_{i} \}$ \quad (\sanityCheck)
 	\item $\relBlock _{i + 1} = \relBlock _{i} + (1 - \locIsCoinbase _{i}) \cdot \locIsCoinbase _{i + 1}$
 \end{enumerate}
 We end on ``finalization'' constraints.
 The \zkEvm{} expects to process blocks, as such we impose these constraints unconditionally.
 \begin{enumerate}[resume]
-	\item $\locIsBasefee  _{N} = 1$
+	\item $\locIsBlobBasefee _{N} = 1$
```

Comment:
I'm unsure why `IsBaseFee` is not relevant here anymore.

Reply:
The reason is that the "cycle" of instruction previously ended on `BASEFEE`, now it ends with `BLOBBASEFEE`. This finalization constraint reflects this.